scripts/itineraries/ItineraryProvider.js

```diff
@@ -0,0 +1,33 @@
+const eventHub = document.querySelector(".container")
+
+let itineraries = []
+
+const dispatchStateChangeEvent = () => {
+    const itineraryStateChangedEvent = new CustomEvent("itineraryChange")
+    eventHub.dispatchEvent(itineraryStateChangedEvent)
+}
+
+
+export const getItinerary = () => {
```

Comment:
Think we can make this plural, i.e. `getItineraries`, just for consistency with our other `getX` functions in our Providers?